src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1264,6 +1264,7 @@ private String getContainerCommand() {
                 httpPortToUse = findAvailablePort(LIBERTY_DEFAULT_HTTP_PORT, false);
                 httpsPortToUse = findAvailablePort(LIBERTY_DEFAULT_HTTPS_PORT, false);
             } catch (IOException x) {
+                debug("IOException trying to findAvailablePort(), using default ports.");
```

Comment:
I was thinking to actually print a user-facing error() and include the exception message.  Perhaps even include the stacktrace in the same error or as a separate debug log statement.  Because if findAvailablePort gives an IOException, then that means either there was no available port, or there was an error when opening a socket.  Either seems like a severe issue, and if the `docker run` command fails afterwards (due to those issues), the exception can help to diagnose the problem.

Reply:
Agreed.